HighLevelGoals.md

```diff
@@ -17,8 +17,7 @@
       other than C/C++.
 3. Design to execute within and integrate well with the *existing*
    [Web platform](Web.md):
-    * maintain the versionless, [feature-tested](FeatureTest.md) and
-      [backwards-compatible](BinaryEncoding.md#backwards-compatibility)
+    * maintain the versionless, [feature-tested](FeatureTest.md) and backwards-compatible (TODO: add backwards compatible section)
```

Comment:
Ah, I just meant a comment in the markdown that won't appear in the rendered version.